docs/integrations/amazon-aws/ec2-cloudwatch-metrics.md

```diff
@@ -135,10 +137,24 @@ To configure a CloudTrail Source, perform these steps:
 2. [Configure DataEvents with CloudTrail](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/using-cloudtrail.html) in your AWS account.
 3. Confirm that logs are being delivered to the Amazon S3 bucket.
 4. Add an [AWS CloudTrail Source](/docs/send-data/hosted-collectors/amazon-aws/aws-cloudtrail-source.md) to Sumo Logic.
-5. While configuring the cloud trail log source, following Field can be added in the source:
-    1. Add an **account** field and assign it a value which is a friendly name / alias to your AWS account from which you are collecting logs. This name will appear in the Sumo Logic Explorer View. Logs can be queried via the “account field”.
-
+	1. **Name**. Enter a name to display the new Source.
+	2. **Description**. Enter an optional description.
+	3. **S3 Region**. Select the Amazon Region for your API Gateway S3 bucket.
+	4. **Bucket Name**. Enter the exact name of your API Gateway S3 bucket.
+	5. **Path Expression**. Enter the string that matches the S3 objects you'd like to collect. You can use a wildcard `*` in this string.
+:::note
+DO NOT use a leading forward slash. See [Amazon Path Expressions](/docs/send-data/hosted-collectors/amazon-aws/Amazon-Path-Expressions). The S3 bucket name is not part of the path. Don’t include the S3 bucket name when you are setting the Path Expression.
+:::
+	6. **Source Category**. Enter aws/observability/cloud trail/logs.
+	7. **Fields**. Add an **account** field and assign it a value that is a friendly name/alias to your AWS account from which you are collecting logs. This name will appear in the Sumo Logic Explorer View. Logs can be queried through the “account field”.
 <img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/AWS-EC2-CW-Metrics/Fields.png')} alt="Fields" />
+	8. **Access Key ID and Secret Access Key**. Enter your [Access Key ID and Secret Access Key](http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSGettingStartedGuide/AWSCredentials.html). Learn how to use Role-based access to AWS [here](/docs/send-data/hosted-collectors/amazon-aws/aws-sources).
+	9. **Log File Discovery -> Scan Interval**. Use the default of 5 minutes. Alternately, enter the frequency. Sumo Logic will scan your S3 bucket for new data. Learn how to configure Log File Discovery [here](/docs/send-data/hosted-collectors/amazon-aws/aws-sources).
+	10. **Enable Timestamp Parsing**. Select the check box.
+	11. **Time Zone**. Select Ignore time zone from the log file and instead use, and select UTC.
```

Comment:
Lines 154, 155: Could you please recheck and confirm if we are conveying the correct instructions?

Reply:
Hi Megha, as these steps are present in 44 apps docs total, [I've created a separate ticket for Nishant](https://app.asana.com/0/1183741177809740/1203999216349981/f) to address this with the Apps and AWSO teams.